### PR TITLE
fixes #11897 - relax rest-client version pinning

### PR DESCRIPTION
**Pull request by mmoll**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,7 +1,10 @@
 source 'https://rubygems.org'
 gemspec
 
+gem 'rest-client', '< 1.7' if RUBY_VERSION == '1.8.7' # 1.7.0 drops support for ruby 1.8.7
+
 group :development do
   gem 'pry'
   gem 'smart_proxy', :github => "theforeman/smart-proxy", :branch => 'develop'
+  gem 'addressable', '~> 2.3.8' if RUBY_VERSION == '1.8.7' # 2.4.0 drops support for ruby 1.8.7
```
Comment (mmoll) [inline Gemfile:9]: this is needed for 1.8.7 tests to go green with current develop

**Comment by mmoll** [comment]
Tests are green, so maybe there's nothing else to do... However, as I don't use discovery at the moment, this is untested with newer rest-client versions by me.

**Comment by lzap** [comment]
Merged as c5c618810925cc7bb31b7eee28c139f38306b394, thank you!

**Comment by domcleal** [comment]
@lzap if you could cut a release of this gem in the near future, it'd be appreciated - we won't be able to package it for Ubuntu 16.04 and F24 otherwise.  (Or update rest-client elsewhere.)

**Comment by lzap** [comment]
```
$ gem push smart_proxy_discovery-1.0.4.gem
Pushing gem to https://rubygems.org...
Successfully registered gem: smart_proxy_discovery (1.0.4)
```

**Comment by domcleal** [comment]
Thanks, RPM updated.

**Pull request closed without merging**